lib/sort.js

```diff
@@ -0,0 +1,25 @@
+export function sort(fn, array) {
```

Comment:
To zadziala ale jest bardzo niewydajne.
Można to zrobić `reduce`'em w pętli `while`